paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -110,9 +112,22 @@ data class DeviceConfig(
         versionQualifier = VersionQualifier()
       }
 
+  private val currentWidth: Int
+    get() = if (orientation == ScreenOrientation.PORTRAIT) {
+      min(screenWidth, screenHeight)
```

Comment:
curious: are there any current usages of `screenWidth` and `screenHeight` that might benefit from now using `currentWidth` and `currentHeight`?  Maybe not, but just checking.

Reply:
I looked through the usages and I don't think we'd need to use these elsewhere yet